src/main/scala/cognite/spark/v1/TimeSeriesRelation.scala

```diff
@@ -132,7 +132,9 @@ case class TimeSeriesUpsertSchema(
     metadata: Option[Map[String, String]] = None,
     assetId: Option[Long] = None,
     description: Option[String] = None,
-    securityCategories: Option[Seq[Long]] = None
+    securityCategories: Option[Seq[Long]] = None,
```

Comment:
Does the order matter? It's quite unsorted, if it matches what we use at doc.cognitedata.com it's ok I suppose. Otherwise id, name, description, metadata should all be closer to the top instead. Not sure if that breaks backwards compatibility but probably worth changing now rather than later even so.